batch/batch/server/server.py

```diff
@@ -62,8 +63,9 @@ def schedule(ttl, fun, args=(), kwargs=None):
 routes = web.RouteTableDef()
 aiohttp_jinja2.setup(app, loader=jinja2.PackageLoader('batch', 'templates'))
 
-db = Database.create_synchronous(os.environ.get('CLOUD_SQL_CONFIG_PATH',
-                                                '/batch-secrets/batch-production-cloud-sql-config.json'))
+loop = asyncio.get_event_loop()
+db = loop.run_until_complete(BatchDatabase(os.environ.get('CLOUD_SQL_CONFIG_PATH',
```

Comment:
I would again stick to either create_synchronous everywhere, this more verbose method. Currently using both. One solution is to just use the `run_synchronous` function directly

Reply:
done.

Reply:
done.